modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -30,7 +30,7 @@ class UploadsController < ApplicationController
       def submit
         Datadog::Tracing.active_trace&.set_tag('form_id', params[:form_number])
 
-        response = if use_itf_api_for_210966_form?
+        response = if intent_service.use_intent_api?
                      handle_210966_authenticated
```

Comment:
I want to move the logic from `handle_210966_authenticated` into the service as well, perhaps in the next refactor PR before I actually tackle the real change with `participant_id`.